core-java-modules/core-java-io2/README.md

```diff
@@ -0,0 +1,6 @@
+=========
+
+## Core Java IO Cookbooks and Examples
+
+### Relevant Articles: 
+- [Reading Files Versus Loading Resources](http://www.baeldung.com/reading-files-versus-loading-resources)
```

Comment:
We don't conventionally add the links to the articles until everything's published. Let's ditch the README.md

Reply:
done